.github/workflows/discuss-notification.yml

```diff
@@ -29,9 +44,14 @@ jobs:
           -H 'Content-Type: application/json' \
           --data-binary @- << EOF
           {
-            "title": "Axon Framework - Release ${{ github.event.release.name }}",
-            "raw": "I’m glad to announce that we’ve released ${{ github.event.release.name }} on ${{ github.event.release.createdAt }}\n\n. This release contains: \n\n ${{ github.event.release.description}}",
+            "title": "${{ release_name }}",
+            "raw": "I’m glad to announce that we’ve released \n\n${{ release_name }}\n\n on \n\n${{ release_date }}\n\n.<br\>
```

Comment:
Will the date be somehow changed to something human readable? I also wonder how much it would add, since the message itself will already have a date.